skymp5-server/cpp/server_guest_lib/MpActor.cpp

```diff
@@ -52,8 +52,12 @@ void MpActor::VisitProperties(const PropertiesVisitor& visitor,
 {
   auto baseId = MpObjectReference::GetBaseId();
   uint32_t raceId = GetLook() ? GetLook()->raceId : 0;
+  BaseActorValues baseActorValues;
+  if (GetParent()->HasEspm()) {
+    auto& espm = GetParent()->GetEspm();
+    baseActorValues = GetBaseActorValues(espm, baseId, raceId);
+  }
```

Comment:
@Pospelove BTW maybe we should log some warning here so it would be clear why values are not expected ones (if someone will encounter this issue sometime not during startup).

Reply:
this is nearly possible, this var is always initialized in real-world usage. but we should I agree. but it also is going to flood in unit tests. maybe we need to create an issue to fix that somehow. it's a project-wide problem. @nic11 
We shouldn't stop the procedure of accepting this PR due to this issue.

Reply:
> We shouldn't stop the procedure of accepting this PR due to this issue.

I absolutely agree, just 'BTW' :)